### PR TITLE
Jenkins courses added in free-courses-en.md

### PR DESCRIPTION
**Pull request by Manish-Syal123**
## What does this PR do?
Add new courses
### Description
Added courses from youtube and some free websites  
### Why is this valuable (or not)?
Jenkins is used to build and test the product continuously, so developers can continuously integrate changes into the build. Jenkins is the most popular open source CI/CD tool on the market today and is used in support of DevOps, alongside other cloud native tools.
### How do we know it's really free?
 As the courses are  from youtube so it is free and the other one is the site that provides free courses in English
### For book lists, is it a book? For course lists, is it a course? etc.
It's a course
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -769,6 +770,16 @@
 * [Vuex for Everyone](https://vueschool.io/courses/vuex-for-everyone)
 
 
+### Jenkins
+
+* [FREE Advanced Jenkins in K8s (Docker in Docker)](https://coursesity.com/course-detail/free-advanced-jenkins-in-k8s-docker-in-docker?ref=share-link) - coursesity (Google)
```
Comment (eshellman) [inline courses/free-courses-en.md:775]: this looks like an Udemy course why "google".
remove tracking attributes from link.

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -769,6 +770,16 @@
...
+* [Getting Started with Jenkins](https://www.simplilearn.com/learn-jenkins-basics-free-course-skillup) - simplilearn (Google)
+* [Jenkins](https://youtube.com/playlist?list=PLhW3qG5bs-L_ZCOA4zNPSoGbnVQ-rp_dG) - Automation Step by Step (YouTube)
+* [Jenkins Full Course | Jenkins Tutorial For Beginners | Jenkins Tutorial | Simplilearn](https://youtu.be/FX322RVNGj4) - Simplilearn (YouTube)
```
Comment (eshellman) [inline courses/free-courses-en.md:778]: For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes
no shortened links

**Review by eshellman — CHANGES_REQUESTED** [review]
Please read docs/CONTRIBUTING.md guidelines!
I've looked at 3 of links; pelease look at the others.

**Review by Manish-Syal123 — COMMENTED** [review]
All the changes have made , Please take a look

**Comment by eshellman** [comment]
please resolve conflicts.

**Comment by Manish-Syal123** [comment]
Resolved conflicts 👍

**Comment by eshellman** [comment]
I think that in resolving conflicts you deleted the newerJulia section

**Review by Manish-Syal123 — COMMENTED** [review]
* [Learning Jenkins (2018)](https://www.linkedin.com/learning/learning-jenkins-14423877?replacementOf=learning-jenkins-2018) (Linkedin)

**Review by Manish-Syal123 — COMMENTED** [review]
added the deleted course Julia again

**Review by Manish-Syal123 — COMMENTED** [review]
* [Julia](#julia)

**Comment by Manish-Syal123** [comment]
forgot to add * [Julia](#julia) in index

File: courses/free-courses-en.md [hunk 3]
```diff
@@ -728,6 +731,8 @@
 * [Node.js Crash Course Tutorial](https://www.youtube.com/playlist?list=PL4cUxeGkcC9jsz4LDYc6kv3ymONOKxwBU) - The Net Ninja
 * [Node.js Tutorial for Beginners](https://www.youtube.com/playlist?list=PLsyeobzWxl7occsESx2X1E2R2Uw5wCoeG) - Telusko (YouTube)
 * [Node.js Tutorial for Beginners: Learn Node in 1 Hour](https://www.youtube.com/watch?v=TlB_eWDSMt4) - Moshfegh Hamedani (Programming with Mosh)
+* [Node.js Tutorials](https://www.youtube.com/playlist?list=PLWKjhJtqVAbmGQoa3vFjeRbRADAOC9drk) - Beau,Pedro Mercado,freeCodeCamp (Youtube)
```
Comment (eshellman) [inline courses/free-courses-en.md:734]:
```suggestion
* [Node.js Tutorials](https://www.youtube.com/playlist?list=PLWKjhJtqVAbmGQoa3vFjeRbRADAOC9drk) - freeCodeCamp.org
```

File: courses/free-courses-en.md [hunk 4]
```diff
@@ -792,6 +797,16 @@
 * [Vuex for Everyone](https://vueschool.io/courses/vuex-for-everyone)
 
 
+### Jenkins
+
+* [FREE Advanced Jenkins in K8s (Docker in Docker)](https://www.udemy.com/course/free-advanced-jenkins-in-k8s-docker-in-docker) - Nomad Sashi (Udemy)
+* [Jenkins](https://youtube.com/playlist?list=PLhW3qG5bs-L_ZCOA4zNPSoGbnVQ-rp_dG) - Raghav Pal (YouTube)
+* [Jenkins Tutorial](https://www.mygreatlearning.com/academy/learn-for-free/courses/jenkins-tutorial) (Greate Learning)
+* [Jenkins Tutorial Step by Step](https://www.youtube.com/playlist?list=PL8VbCbavWfeGt_Hyq0h9Pymi9DTGowd5X) - Pramod Dutta (YouTube)
```
Comment (eshellman) [inline courses/free-courses-en.md:805]:
```suggestion
* [Jenkins Tutorial Step by Step](https://www.youtube.com/playlist?list=PL8VbCbavWfeGt_Hyq0h9Pymi9DTGowd5X) - Pramod Dutta
```

**Review by eshellman — CHANGES_REQUESTED** [review]

**Pull request closed without merging**